docs/sdk/client-side-sdks/android/android-gettingstarted.md

```diff
@@ -9,57 +9,60 @@ sidebar_custom_props: { icon: material-symbols:rocket }
 [![Maven](https://badgen.net/maven/v/maven-central/com.devcycle/android-client-sdk)](https://search.maven.org/artifact/com.devcycle/android-client-sdk)
 [![GitHub](https://img.shields.io/github/stars/devcyclehq/android-client-sdk.svg?style=social&label=Star&maxAge=2592000)](https://github.com/DevCycleHQ/android-client-sdk)
 
+[//]: # (wizard-initialize-start)
```

Comment:
hmm does this mean we're going with java over kotlin for android?

Reply:
I think all new Android projects use Kotlin now, and its the default in their own docs I think?

Reply:
actually looks like its more of a toss up in their docs, most of the time theres a choice of java or kotlin

Reply:
hmm we could always just include both, if Cobi's cool with that

Reply:
i guess in the meantime i'll switch it to kotlin